core/src/main/java/hudson/model/Queue.java

```diff
@@ -877,6 +877,13 @@ private void _getBuildableItems(Computer c, List<BuildableItem> col, List<Builda
     }
 
     /**
+     * Gets the snapshot of all {@link BlockedItem}s.
+     */
+    public List<BlockedItem> getBlockedItems() {
```

Comment:
@olivergondza noted in the commit that this should not be public as it is for testing only